packages/dataviews/src/field-types/integer.tsx

```diff
@@ -4,6 +4,9 @@
 import type { SortDirection, ValidationContext } from '../types';
 
 function sort( a: any, b: any, direction: SortDirection ) {
+	if (a === null) return 1;
+	if (b === null) return -1;
```

Comment:
Why do we have to handle "null" values in "integer" type. Null is not an integer (same question for text)

Reply:
The field type doesn't do "data typing", it's just a way for us to provide default implementations of certain functions. As a result, we may end up having data that is string or null. Another way to put it: how should we work with types that can be nullable?

A related conversation was this: given that data can be null, it'd be interesting to having a precreated filter (`elements` not required) that allows users to filter by "empty data / not empty data". Similar to what a spreadsheet has:

https://github.com/user-attachments/assets/6c343e7a-53a1-4caa-8c42-502b0f9f9770

Reply:
> Another way to put it: how should we work with types that can be nullable?

Should we allow this? That would be my first question. Maybe it's ok for "null/undefined" values but for me it's not for say assigning strings to numbers for instance. And we should consider that "doing it wrong" rather than actually providing a fix.

Reply:
I see, and I agree to that. Looking at the current use cases of the WP table in the wp-admin, and considering this will be used with user data, I'd think we need to support nullable data. I was surprised we hadn't yet run into this, to be honest.

Reply:
I'm on the fence but I wonder if:

 - If type is "string/text", treat empty values as ""
 - If type is "number", treat it as 0
 - If there's no type, move to the end or something.

Reply:
For strings, I don't really see why "" would be different than `null` or `undefined`. For the rest I don't mind much.

Reply:
To clarify, I also don't mind any approach to be honest.